modules/tinymce/CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Changed the load order so that the content css gets loaded before the editor gets populated with contents #TINY-7249
 - Changed `emoticons`, `wordcount`, `code`, `codesample`, and `template` plugins to open dialogs using commands #TINY-7619
+- Changed `formatter.match` to take an optional `similar`-parameter #TINY-7712
```

Comment:
```suggestion
- Changed `formatter.match` to take an optional `similar` parameter #TINY-7712
```

Reply:
This should also be in the improved list, not in the changed list.